src/RubberDuckyParking/parking_api/urls.py

```diff
@@ -5,9 +5,9 @@
 from . import views
 
 urlpatterns = [
-    path('', views.invalid_parameters, name='invalid_parameters'),  # invalid request
+    path(r'^$', views.test, name='invalid_parameters'),  # invalid request
```

Comment:
The django website suggested r"^$' instead of '' because its the character code for an empty string

Reply:
I still don't see how much better this works